### DISCUSSION TITLE
Filter Telefonnummer

### DISCUSSION THREAD

**Discussion by hergie1976** [der8en/Fritz-Call-Monitor-Feed#10] [Q&A]
Hallo,

was muss am Script geändert werden, damit das Telefonbuch nur für eine Telefonnummer gefüllt wird?

Wir haben über VoIP 3 Telefonnummern, von der mich nur eine interessiert. Wie sieht der Filter aus, bzw. was muss gemacht werden?

Gruß

Heiko

**Comment by hergie1976** [comment]
Habe noch keine Lösung.

Das state_attr wechselt die Daten von "dialing" bzw. "ringing" zu "talking".
Bei "dialing" bzw. "ringing" habe ich noch die Daten "from" bzw. "to", in denen der Ursprung steht, also die Telefonnummer, auf die ich filtern will. Wechselt es zu "talking" sind die Daten "from" bzw. "to" nicht mehr vorhanden.

Gerade finde ich kein Weg.

**Comment by Dtrieb** [comment]
Dann speicher dir doch die Ursprungsnummer in einem Helper (Zahl) dann kannst du danach nur auslösen lassen.
